template/action_execute_code.go

```diff
@@ -69,7 +70,7 @@ func (t *WorkatoTemplate) getExecuteCode(service *gendoc.Service, method *gendoc
 				}
 			}
 
-			if mthd == httpGet && hasRepeatedType(t.messageMap[method.RequestFullType]) {
+			if (mthd == httpGet || mthd == httpDelete) && hasRepeatedType(t.messageMap[method.RequestFullType]) {
```

Comment:
Is there any reason this needs to be restricted to certain HTTP verbs?

Reply:
not 100% reason. Just to contain for what we need (GET and DELETE).
I assume POST/PUT/PATCH they usually take BODY and people don't bother with query params.
I can remove that restriction if you want. Will not make a difference (probably ... need to check)

Reply:
Yeah seems weird to have an arbitrary restriction with no real reason. As we are just going to have to remove it later when we do find a use case.

Reply:
fair enough. I removed them